README.md

```diff
@@ -103,13 +118,38 @@ Sample setting:
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
 
+## Requirements 
```

Comment:
In what sense are these `Requirements` different from the earlier `Prerequisites`?
Should this be moved into the `Developer's Guide` section?

Reply:
@echanlee Can you address these two comments so that we can merge this PR?

Reply:
I will move this to the `Requirements` section